tests/integration/test_fim/test_registry/test_recursion_level/data/wazuh_recursion_windows_registry.yaml

```diff
@@ -7,22 +8,18 @@
     - disabled:
         value: 'no'
     - windows_registry:
-        value: "HKEY_LOCAL_MACHINE\\SOFTWARE\\norecursion"
+        value: REGISTRY_0
         attributes:
-        - recursion_level: "0"
-        - FIM_MODE
+        - recursion_level: LEVEL_0
     - windows_registry:
-        value: "HKEY_LOCAL_MACHINE\\SOFTWARE\\recursion1"
+        value: REGISTRY_1
         attributes:
-        - recursion_level: "1"
-        - FIM_MODE
+        - recursion_level: LEVEL_1
     - windows_registry:
-        value: "HKEY_LOCAL_MACHINE\\SOFTWARE\\recursion5"
+        value: REGISTRY_2
         attributes:
-        - recursion_level: "5"
-        - FIM_MODE
+        - recursion_level: LEVEL_2
     - windows_registry:
-        value: "HKEY_LOCAL_MACHINE\\SOFTWARE\\recursion512"
+        value: REGISTRY_3
         attributes:
-        - recursion_level: "512"
-        - FIM_MODE
+        - recursion_level: LEVEL_3
```

Comment:
You are creating the registry keys with a 64 bit architecture, but the configuration does not set the architecture, being 32 bit by default. This causes the test to fail. Add an `arch` attribute to fix this.